design/one-pager-trait-workload-interaction-mechanism.md

```diff
@@ -0,0 +1,179 @@
+# Traits and workloads interaction mechanism in OAM
+
+* Owner: Ryan Zhang (@ryanzhang-oss)
+* Reviewers: Crossplane Maintainers
+* Status: Draft
+
+## Terminology
+
+* **CRD (Custom Resource Definition)** : A standard Kubernetes Custom Resource Definition
+* **CR (Custom Resource)** : An instance of a Kubernetes type that was defined using a CRD
+* **GVK (Group Version Kind)** : The API Group, Version, and Kind for a type of Kubernetes
+ resource (including CRDs)
+* **Workload child resources** : The Kubernetes resources generated by a workload controller. They
+ should all have a owner reference pointing to the parent workload instance.
+
+## Background
```

Comment:
The background section is way too simplified. It should also cover some user scenarios as discussed in https://github.com/crossplane/crossplane/issues/1448#issuecomment-620323239.